artio-codecs/src/test/resources/uk/co/real_logic/artio/dictionary/example_dictionary.xml

```diff
@@ -1,5 +1,5 @@
 <!-- Simple Data Dictionary for testing purposes -->
-<fix major="7" minor="2">
+<fix type="FIXR" major="7" minor="2">
```

Comment:
What does the DictionaryParser do if this type attribute is missing, as it will be on older XML files.